### PR TITLE
Dependabot!

### PR DESCRIPTION
**Pull request by quad**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: weekly
+
+  - package-ecosystem: gomod
+    directory: /
+    schedule:
+      interval: weekly
```
Comment (boblail) [inline .github/dependabot.yml:16]: The Go implementation has zero dependencies

**Review by boblail — APPROVED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (quad) [reply .github/dependabot.yml:16]: BELT AND SUSPENDERS BABEEEE

Comment (quad) [reply .github/dependabot.yml:16]: Happy to remove?

Comment (boblail) [reply .github/dependabot.yml:16]: No, you can keep it!

**Pull request merged**